src/settings.ts

```diff
@@ -53,6 +53,10 @@ export interface IEditorServiceSettings {
 export interface IFormatSettings {
   enable?: boolean;
 }
+export interface IHoverSettings {
+  showMetadataInfo?: boolean;
+  // showPuppetfileInfo?: boolean; Future use
```

Comment:
comment code delete?

Reply:
This is to show intention for more hover features to configure